lib/core/src/msParam.cpp

```diff
@@ -1022,6 +1022,28 @@ parseMspForPosInt( msParam_t* inpParam ) {
     return myInt;
 }
 
+int parseMspForStr_new(msParam_t* inpParam, char** outParam)
+{
+    if (inpParam == NULL || inpParam->inOutStruct == NULL) {
+        rodsLog(LOG_ERROR, "%s: inpParam type %s is not STR_MS_T", __func__, inpParam->type);
```

Comment:
Let's use the new logger (i.e. `rodsLog` is going to be removed eventually).

I'm pretty sure the new logger is used in other functions within this file. Look for `log_msi` within this file to find example usage.

_This applies to the other locations too._